examples/ssr-caching/app/page.tsx

```diff
@@ -0,0 +1,16 @@
+export const revalidate = 10;
```

Comment:
This is changing the behavior of what the example was originally showing. Originally the example was showing caching at the CDN layer via `Cache-Control` headers alone, using `export const revalidate` opts the page into ISR which has build time level caching and other features to it. 

We can still convert this to app router although it requires making this page dynamic via `export const dynamic = 'force-dynamic'` and then setting the `Cache-Control` header in `next.config` or `middleware`.

Reply:
@PapatMayuri Actually to clarify, neither `next.config` and Next.js Middleware work for overriding the `Cache-Control` header in App Router. We'll have to wait for one of those options to be fixed before proceeding with this PR.

Reply:
We can land this now that https://github.com/vercel/next.js/pull/69802 has landed

Reply:
Thanks, @ijjk I will Update this PR tomorrow Morning.

Reply:
@samcx I have updated PR as per the comment please check.